### ISSUE TITLE
meta-data `com.onesignal.suppressLaunchURLs` does not work

### ISSUE THREAD

**Issue by litmon** [OneSignal/OneSignal-Android-SDK#1312]
**Description:**

I set meta-data `com.onesignal.suppressLaunchURLs` to `true`, but it doesn't work as expected.
https://documentation.onesignal.com/docs/links

In my investigation, it happened that `OSUtils.getManifestMeta(context, "com.onesignal.suppressLaunchURLs")` returns `null`, because meta-data is set to as boolean not String in Bundle.
https://github.com/OneSignal/OneSignal-Android-SDK/blob/87ff8245c974d3e6b0349a7c9eeab23b4b251736/OneSignalSDK/onesignal/src/main/java/com/onesignal/OSUtils.java#L405-L409

```java
bundle.getString('com.onesignal.suppressLaunchURLs') //=> null
bundle.getBoolean('com.onesignal.suppressLaunchURLs') //=> true
```

So `launchUrlSuppress` always set to false.

https://github.com/OneSignal/OneSignal-Android-SDK/blob/87ff8245c974d3e6b0349a7c9eeab23b4b251736/OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java#L2160

**Environment**

Android SDK version: 4.3.0

**Steps to Reproduce Issue:**

1. Add meta-data tag in AndroidManifest.xml
1. Receive a push notification with launchURL, and open

```
<meta-data android:name="com.onesignal.suppressLaunchURLs" android:value="true" />
```

**Anything else:**

(crash stacktraces, as well as any other information here)

**Comment by Jeasmine** [comment]
Hey @litmon, sorry for the late response, nice catch! That is indeed what is happening, and we are working on solving this. Thanks again for pointing this up.

**Comment by jkasten2** [comment]
@litmon This is now fixed in the [4.4.0 release](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/4.4.0)
